### PR TITLE
Fix Bug 1131331 - Update meta tags on mozilla.org homepage

### PR DESCRIPTION
**Pull request by kyoshino**
There are 2 new strings behind the `meta_update_201403` tag. Cc @flodolo

### CODE REVIEW

**Comment by kyoshino** [comment]
There are 2 new strings. Cc @flodolo

File: bedrock/mozorg/templates/mozorg/home/home-new.html [hunk 1]
```diff
@@ -6,7 +6,22 @@
 
 {% add_lang_files "mozorg/home/index" %}
 
-{% block page_title %}{{_('Home of the Mozilla Project')}}{% endblock %}
+{% block page_image %}{{ static('img/home/page-image.png') }}{% endblock %}
+
+{% block page_title %}
+  {% if l10n_has_tag('meta_update_201403') %}
```
Comment (flodolo) [inline bedrock/mozorg/templates/mozorg/home/home-new.html]: 201503 ;-)

Comment (kyoshino) [inline bedrock/mozorg/templates/mozorg/home/home-new.html]: aha :blush: fixed!

**Comment by alexgibson** [comment]
This is an r+ from me - ok to merge this one now @pascalchevrel or @flodolo?

**Comment by flodolo** [comment]
Yes, good to merge. Since this is not urgent and behind flags, I'm going to expose the strings later this week together with other updates.

**Pull request merged**